src/ext/dispatch.c

```diff
@@ -213,215 +199,97 @@ static void execute_fcall(ddtrace_dispatch_t *dispatch, zval *this, zend_execute
     Z_DELREF(closure);
 }
 
-static int is_anonymous_closure(zend_function *fbc, ddtrace_lookup_data_t *lookup) {
```

Comment:
👍 great stuff and getting rid of cargoculted checks! 
For long time I assumed that odd `{closure}` function can slip without ZEND_ACC_CLOSURE being properly set.
Thanks to your PR and our testsuite just verified that assumption to be false - by adding an assertion on the name and running tests: https://circleci.com/workflow-run/48f8d03a-cde4-45b7-9a3a-a9d88909b264
Good stuff!